compiler/linter.nim

```diff
@@ -10,8 +10,9 @@
 ## This module implements the style checker.
 
 import strutils
+from sugar import dup
```

Comment:
```suggestion
from std/sugar import dup
```

Reply:
I'll do it if another change is required, otherwise it can be batched with similar changes now that https://github.com/nim-lang/Nim/pull/17902 allows this